tutorials/physics/interpolation/advanced_physics_interpolation.rst

```diff
@@ -0,0 +1,173 @@
+.. _doc_advanced_physics_interpolation:
+
+Advanced physics interpolation
+==============================
+
+Although the previous instructions will give satisfactory results in a lot of games,
+in some cases you will want to go a stage further to get the best possible results
+and the smoothest possible experience.
+
+Exceptions to automatic physics interpolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'm not sure if it matters for rendering the page, but at least for consistency, we should use the order for header syntax that is standard in other pages:
```
Page title
==========

Header
------

Sub-header
~~~~~~~~~~

Sub-sub-header
^^^^^^^^^^^^^^
```

This page uses `^^^` for everything, so all those should use `---` instead. And you should consider using subheaders with `~~~` for some of the headers below `Cameras`, since they're all meaningfully related. UNLESS you are intentionally trying to keep everything at the same header level so it shows up in the sidebar.

The other pages here need similar changes. If you want specific suggestions for exactly which header syntax to use for each I can make individual suggestions for them.

Reply:
This can also be done in a later PR without breaking translations, so feel free to ignore and defer this one

Reply:
I'll implement this, and change all the ``^^^^`` to ``~~~~``